tools/theme/colors.ts

```diff
@@ -9,8 +9,9 @@ const colors = {
   secondary: {
     critical: '#DF1414',
     criticalHover: '#B21010',
+    successHover: '#1e7e34',
     warning: '#B98900',
-    success: '#377E62',
+    success: '#009444',
```

Comment:
Changed this to match the color specified in the figma design system
![image](https://user-images.githubusercontent.com/51224641/135763900-0ba70bad-d5b2-47c3-b6da-78224be7769e.png)